src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/TransactionTypeMapper.java

```diff
@@ -0,0 +1,20 @@
+package uk.nhs.digital.nhsconnect.nhais.mapper;
+
+import uk.nhs.digital.nhsconnect.nhais.exceptions.ParameterValidationException;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.ReferenceTransactionType;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class TransactionTypeMapper {
+
+    public ReferenceTransactionType.Outbound mapTransactionType(String transactionTypeParam) {
+        switch(transactionTypeParam) {
+            case "$nhais.acceptance": return ReferenceTransactionType.Outbound.ACCEPTANCE;
+            case "$nhais.removal": return ReferenceTransactionType.Outbound.REMOVAL;
+            case "$nhais.deduction": return ReferenceTransactionType.Outbound.DEDUCTION;
```

Comment:
@mattd-kainos Why do we use those weird `$nhais.` prefixes?

Reply:
The requests are nominally FHIR Operations - but we're far from being a FHIR server! https://fhir-drills.github.io/operations.html

Reply:
Ok I understand `$` but do we need `nhais.`?

Reply:
differentiates it from other demographics services with similar concepts